titus-server-runtime/src/main/java/com/netflix/titus/runtime/connector/eviction/replicator/GrpcEvictionReplicatorEventStream.java

```diff
@@ -53,7 +53,10 @@ public GrpcEvictionReplicatorEventStream(EvictionServiceClient client,
     @Override
     public Observable<ReplicatorEvent<EvictionDataSnapshot>> connect() {
```

Comment:
all these `connect` methods seem very similar. Consider factoring them out?

Reply:
+1
I think it is the right time to do it.